controllers/imageclusterinstall_controller_test.go

```diff
@@ -1518,7 +1519,7 @@ var _ = Describe("Reconcile", func() {
 		Expect(infoOut.MachineNetwork[0].CIDR.String()).To(Equal(clusterInstall.Spec.MachineNetwork))
 	})
 
-	It("in case there is no actual bmh under the reference we should not return error", func() {
+	It("in case there is no actual bmh under the reference we should return error", func() {
```

Comment:
Do we really need both this test and `Set ClusterInstallRequirementsMet to false in case there is not actual bmh under the reference`?